Resources/Templates/dancer.yml.twig

```diff
@@ -0,0 +1,139 @@
+#
+# overwrite: ask
+
+# Once interactive mode is disabled, the system needs to know which profile to use when.
+# Fortunately SkeletonDancer provides two ways to automate this:
+#
+# pattern-map: Use an associative array where the key is a relative patch like 'Bundles'
+#              or a complete regex. See http://symfony.com/doc/current/components/finder.html#path
+#              for more information. And the value is the profile-name. Eg: `{ 'Bundles/': symfony-bundle }`
+#
+# Custom class: Specify an autoloadable class (see autoloading section below)
+#               with a `resolve()` method that receives the current folder-name (relative)
+#               and a `Rollerworks\Tools\SkeletonDancer\Configuration\Config` instance.
+#               And returns the resolved profile-name. Eg. "Acme\\SkeletonDancer\\ProfileResolver".
+#
+# Note: The path is relative to the project's root-folder, not the current directory.
+#
+# profile-resolver: ~
+
+# Some profiles share the same generators (and configurators).
+# Instead of having to duplicate all values for all profiles you can
+# specify the shared defaults here.
+#
+# Tip: Defaults are used as they come, you can define custom defaults
+# that are only used by other defaults (using the expression language)
+# and never used by configurators.
+#
+# A value can be a constant (as-is) or start with an `@` to be eveluated as an expression
+# http://symfony.com/doc/current/components/expression_language/syntax.html
+#
+# Caution: variables are resolved by order, you cannot reference a variable
+# that's defined later in the list.
+#
+# Examples:
+# foo: "my_default_value" # Value used as-is, no expression used.
+# bar: "@foo" # uses the value of variable `foo` as-is
+# peep: "@@foo" # will be transformed to `@foo`, and will not evaluated as an expression
+# foobar: "@substr(foo, 0, 2)" # will run variable `foo` trough the substr() function
+#
+# Expressions are a powerful tool to make your defaults more dynamic.
+# To get a value from the configuration use the `get_config()` function like:
+# namespace: "@'Acme\\Application\\' ~ ucfirst(get_config('current_dir_name'))"
+#
+# defaults:
+{% if shared_defaults is not empty %}
+#    {{ shared_defaults|yaml_dump|indent_lines|yaml_comment }}
+{%- endif %}
+
+# SkeletonDancer already comes pre-bundled with many profiles.
+# But it's also possible to define your own, or overwrite existing profiles.
+#
+# Note: If you only need overwrite a small portion it may be better to overwrite
+# the template rather then creating a completely new generator.
+#
+# profiles:
+#     # Name of the profile, in lowercase
+#     my-profile:
+#         # Give a nice description about what this profile does.
+#         description: Generates the directory structure of a custom module.
+#         # Optionally import other profiles
+#         # import: [ composer-package ]
+#
+#         # Defaults for this profile, overwrites shared defaults
```

Comment:
This sentence seems strange, better would be `The defaults of this profile. Merges with shared defaults and defaults from imported profiles (for this profile only).`